locales/it-IT.json

```
@@ -639,7 +639,8 @@
     "poll": {
       "count": "{0} voti|{0} voto|{0} voti",
       "ends": "termina {0}",
-      "finished": "terminato {0}"
+      "finished": "terminato {0}",
+      "update": "Aggiorna sondaggio",
```

There was extra `,` at the end.
```suggestion
      "update": "Aggiorna sondaggio"
```